doc/api/net.md

```diff
@@ -632,6 +632,8 @@ case of errors (parse error or so).
 If `exception` is specified, an [`'error'`][] event will be emitted and any
 listeners for that event will receive `exception` as an argument.
 
+Returns `socket`.
```

Comment:
In other areas of the docs, the return is listed as a bullet point along with the arguments. This should follow that convention.